mscore/stafftextproperties.cpp

```diff
@@ -185,7 +185,7 @@ StaffTextProperties::StaffTextProperties(const StaffTextBase* st, QWidget* paren
             QTreeWidgetItem* item = new QTreeWidgetItem(channelList);
             item->setData(0, Qt::UserRole, i);
             if (a->name().isEmpty() || a->name() == Channel::DEFAULT_NAME)
-                  item->setText(0, tr(Channel::DEFAULT_NAME));
```

Comment:
this never worked apparently, that string ('normal', prior to this PR) never made it to Transifex